### ISSUE TITLE
Alexa Guard is not working in 0.103.0

### ISSUE THREAD

**Issue by wictorwilen** [custom-components/alexa_media_player#478]
**Describe the bug**
After installing Home Assistant `0.103.0` the Alexa Guard is no longer working. I get this error in the log:

```
Setup of platform alexa_media is taking longer than 60 seconds. Startup will proceed without waiting any longer.
```

Under Integrations the following is shown:
![image](https://user-images.githubusercontent.com/5854279/70722335-62e69580-1cf7-11ea-9d60-65f0c43f3f6b.png)

And when using the entity on the dashboard it says that it is not available:
![image](https://user-images.githubusercontent.com/5854279/70722368-75f96580-1cf7-11ea-9940-445aa70b04da.png)

**To Reproduce**

- Install Home Assistant 0.103.0

**Expected behavior**
Expect to have access to the Alexa Guard entity

**System details**
 - Home-assistant (version): 0.103.0
 - Hassio (Yes/No): Yes.
 - alexa_media  (version from `const.py` or HA startup): 2.3.4
 - alexapy (version from `pip show alexapy` or HA startup):

**Logs**
See above

**Additional context**

**Comment by wictorwilen** [comment]
The fix in #474 fixes it

**Comment by alandtse** [comment]
Thanks for reporting. Closing since you've identified the duplicate.
